test/aave-v3/TestSupplyVault.t.sol

```diff
@@ -516,7 +523,7 @@ contract TestSupplyVault is TestSetupVaults {
         poolTokens[0] = aDai;
 
         vaultSupplier1.depositVault(daiSupplyVault, amount);
-        vm.roll(block.number + 100);
+        vm.warp(block.timestamp + 10 days);
```

Comment:
As I commented for compound, deposit are not done to the right account

Reply:
I don't understand your comment as this deposit is made to query the expected rewards amount (so it is indeed deposited to the correct account)

Reply:
Ok I think I don't understand why this line is here: 
`vaultSupplier1.depositVault(daiSupplyVault, amount);`

Without we're testing what we want right?

Reply:
the vault's supplier1 deposits `amount` into the vault, which allows us to query the amount of rewards the vault is entitled to after 10 days. When the vault's supplier 2 supplies on behalf of vault's supplier 1, the vault's supplier 1 should be entitled to the same amount of rewards as the vault, ie they should be entitled to all rewards claimable by the vault and the supplier 2 should not accrue rewards. This is what we're testing inside this test

Reply:
Without this line, the vault is entitled to no rewards after 10 days because the vault's supply is near zero

Reply:
Ok but do we agree that we could just have supplier2 depositing on behalf of supplier1, wait a bit of time and just check that supplier1's rewards have accrued?

Reply:
Correct

Perhaps a more advanced version of the current test could be to test that the accrued amount after 20 days for supplier1 is 3 times the amount of rewards claimable at the time supplier 2 supplies on behalf of supplier 1
@pakim249CAL